packages/oid4vci-demo-frontend/src/components/SSIPrimaryButton/index.tsx

```diff
@@ -7,29 +7,46 @@ export interface IButtonProps {
     className?: string,
     disabled?: boolean,
     caption: string,
+    color?: string
     onClick?: () => void
 }
 
 const SSIPrimaryButton: React.FC<IButtonProps> = (props: IButtonProps) => {
     const ecosystem = useEcosystem()
     const config = ecosystem.getComponentConfig<SSIPrimaryButtonConfig>('SSIPrimaryButton')
     const mainContainerStyle = config.styles.mainContainer;
+    const buttonStyle = config.styles.button
     const {caption, disabled = false, onClick, style} = props
-    // TODO text
-    return (
-        <button style={{
-            height: 42,
-            width: 300,
-            borderRadius: 6,
-            background: `${mainContainerStyle.backgroundColor}`,
+    const color = props.color ?? '#FBFBFB'
```

Comment:
for future reference, i like to deconstruct the props so you do not have to type props.x everywhere. like line 19. this also allows to set default like this color.

const {caption, disabled = false, onClick, style, color = '#FBFBFB'} = props

this way you have everything in one place

Reply:
fixed